docs/using/emr.md

```diff
@@ -7,4 +7,124 @@ nav_order: 6
 has_children: false
 ---
 
-# Using lakeFS with EMR
\ No newline at end of file
+# Using lakeFS with EMR
+
+[Amazon EMR](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-what-is-emr.html) is a managed cluster platform that simplifies running big data frameworks, such as Apache Hadoop and Apache Spark.
+
+## Configuration 
+In order to configure Spark on EMR to work with lakeFS we will set the lakeFS credentials and endpoint in the appropriate fields.
+The exact configuration keys depends on the application running in EMR, but their format is of the form:
+    
+lakeFS endpoint: ```*.fs.s3a.endpoint``` 
+
+lakeFS access key: ```*.fs.s3a.access.key```
+
+lakeFS secret key: ```*.fs.s3a.secret.key```
+
+EMR will encourage users to use s3:// with Spark as it will use EMR's proprietary driver. Users need to use s3a:// for this guide to work.
+{: .note}
+
+The Spark job reads and writes will be directed to the lakeFS instance, using the [s3 gateway](../architecture.md#S3 Gateway).
+
+There are 2 options for configuring an EMR cluster to work with lakeFS:
+1. When you create a cluster - All steps will use the cluster configuration.
+   No specific configuration needed when adding a step.
+2. Configuring on each step - cluster is created with the default s3 configuration.
+   Each step using lakeFS should pass the appropriate config params.
+
+## Configuration on cluster creation 
+Use the below configuration when creating the cluster. You may delete any app configuration which is irrelavant for your use-case.
+ ```json
+[
+  {
+    "Classification": "presto-connector-hive",
+    "Properties": {
+      "hive.s3.aws-secret-key": "$LAKEFS_SECRET_KEY",
+      "hive.s3.aws-access-key": "$LAKEFS_SECRET_ACCESS_KEY",
```

Comment:
I think the variables are reversed (i.e. `$LAKEFS_SECRET_ACCESS_KEY` should probably be `hive.s3.aws-secret-key`?)